shuffle/move/examples/e2e/message.test.ts

```diff
@@ -89,7 +90,7 @@ Deno.test("Advanced: Ability to set message from nonpublishing account", async (
     secondUserContext,
     scriptFunction,
     [],
-    ["invoked script function from nonpublishing account"],
+    [mv.Ascii("invoked script function from nonpublishing account")],
```

Comment:
I think the problem with this naming pattern is: it does not match the script function argument type, as it says `Ascii`, but the type is `[]byte`, how I decide when to use `Ascii` and `Raw`?

Reply:
That's a larger question we've touched on a few times. cc @tzakian, @tnowacki 

The script argument type is []byte or vector<u8>, and in the implementation, the script function will decode the bytes to ascii. This was landed in https://github.com/diem/diem/pull/9554.

```rust
fun somefunc(bytes: vector<u8>) {
  let str = ASCII::string(bytes);
}
```

In the client, however, we still need to send the correct bytes, which involves encoding a string as hex. And the only way the TS SDK knows to encode a string as hex, is if the dev tells us to, because they have insight that the script function they are calling wants a string and subsequently calls `ASCII:string(bytes)`. 

In the near or medium term, we will not have a `string` type for an argument to script function, so this is our current solution.

Reply:
I agree with Dimitri, and that this is the right way for now -- the programmer will need to know about what the script is expecting of those bytes in order to set the encoding correctly (which, while not ideal, seems somewhat reasonable and better than sending random bytes in). Eventually we may also have annotations so the ABIs can hold hints about the encoding of those bytes that this can pick up.

Reply:
@xli Is it the language that's throwing us off, because it's technically incorrect? Since ASCII isn't technically a move type, but a helper method.

change Ascii to AsciiBytes? LMK. Keep in mind I'm hoping to land soon bc of next week.
```typescript
[mv.AsciiBytes("invoked script function from nonpublishing account")]
```